dist/t/spec/features/0010_authentification_spec.rb

```diff
@@ -1,17 +1,22 @@
 require "spec_helper"
 
-RSpec.describe "Sign Up & Login" do
-  it "should be able to sign up successfully and logout" do
+RSpec.describe "Authentification" do
```

Comment:
Being this file in `features` subdirectory, wouldn't it be better like this?

`RSpec.feature "Authentification", :type => :feature do`

And also in the other files too.